lib/rules/max-nesting-depth/README.md

```diff
@@ -148,6 +148,145 @@ a {
 }
 ```
 
+### `ignore: ["pseudo-classes"]`
+
+Ignore pseudo-clases.
```

Comment:
We should be really explicit with what is happening here as the behaviour might not be intuitive to all users. How's about?:

```suggestion
Ignore rules where the first selector in each selector list item is a pseudo-class
```